npm/src/clientfilemetadataupload/index.ts

```diff
@@ -3,22 +3,24 @@ import { GraphqlClient } from "../graphql"
 import { IFileMetadata } from "@nationalarchives/file-information"
 
 import {
-  AddFiles,
-  AddClientFileMetadata,
-  AddFilesMutation,
-  AddFilesMutationVariables,
-  AddClientFileMetadataInput
+  AddFilesAndMetadata,
+  AddFilesAndMetadataMutation,
+  AddFilesAndMetadataMutationVariables,
+  StartUpload,
+  StartUploadMutation,
+  StartUploadMutationVariables,
+  ClientSideMetadataInput
 } from "@nationalarchives/tdr-generated-graphql"
 
 import { FetchResult } from "@apollo/client/core"
 import { ITdrFile } from "../s3upload"
 import { FileUploadInfo } from "../upload/upload-form"
 
-declare var METADATA_UPLOAD_BATCH_SIZE: number
+declare var METADATA_UPLOAD_BATCH_SIZE: string
```

Comment:
It's technically another ticket TDR-1232 but I was changing the `createMetadataInputBatches` function anyway so I sneaked the fix for that in as well. I can split it out into another PR if it's easier though.

Reply:
Ah, I see, thanks. It's fine - it would have been easier to review separately, but I've been through the rest of the code now. I was mostly worried it had come in from some merge conflict.